tests/spec/core/issues-notes-spec.js

```diff
@@ -160,6 +160,31 @@ describe("Core — Issues and Notes", function() {
     );
   });
 
+  it("should remove closed issues from spec", function (done) {
+    const githubConfig = {
+      github: {
+        repoURL: "https://github.com/w3c/respec"
+      }
+    };
+    const ops = {
+      config: githubConfig,
+      body:
+        makeDefaultBody() +
+        "<div class='issue' data-number='1530'></div>" +
+        "<section id='issue-summary'></section>",
+    };
+    makeRSDoc(
+      ops,
+      function(doc) {
+        const issueDiv = doc.querySelector("#issue-1530");
+        expect(issueDiv).toBeFalsy();
+        const summarySection = doc.querySelector("#issue-summary");
```

Comment:
I would prefer `doc.getElementById("issue-summary")` as it skips parsing selector syntax, but not a big problem.

Reply:
done :+1:

Reply:
@marcoscaceres, any suggestions? A lot of changes have occurred due to indentation of the code.

Reply:
Don't worry about the indentation for now... worst case, I can run prettier on "develop" and you just rebase on-top. Then things should match.